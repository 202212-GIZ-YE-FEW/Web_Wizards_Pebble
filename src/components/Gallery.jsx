import Image from "next/image";

import gallery from "~/landing/gallery.svg";

export default function Gallery() {
    return (
        <section className='container mt-40 mb-20'>
            <Image
                src={gallery}
                width={1440}
                height={790}
                alt='Volunteers Gallery'
            ></Image>
        </section>
    );
}
