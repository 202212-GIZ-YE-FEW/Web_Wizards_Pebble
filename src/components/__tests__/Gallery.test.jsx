import renderer from "react-test-renderer";

import Gallery from "@/components/Gallery";

it("renders correctly", () => {
    const tree = renderer.create(<Gallery />).toJSON();
    expect(tree).toMatchSnapshot();
});
