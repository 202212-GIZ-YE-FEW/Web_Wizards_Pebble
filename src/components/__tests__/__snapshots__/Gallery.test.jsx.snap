// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="container mt-40 mb-20"
>
  <img
    alt="Volunteers Gallery"
    data-nimg="1"
    decoding="async"
    height={790}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg.jpg&w=3840&q=75"
    srcSet="/_next/image?url=%2Fimg.jpg&w=1920&q=75 1x, /_next/image?url=%2Fimg.jpg&w=3840&q=75 2x"
    style={
      Object {
        "color": "transparent",
      }
    }
    width={1440}
  />
</section>
`;
